independent-publisher-connectors/Dexcom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,145 @@
+                  "type": "string",
+                  "description": "unit"
+                },
+                "rateUnit": {
+                  "type": "string",
+                  "description": "rateUnit"
+                },
+                "egvs": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "systemTime": {
+                        "type": "string",
+                        "description": "Time according to the system clock at which observation was made; nominally UTC"
+                      },
+                      "displayTime": {
+                        "type": "string",
+                        "description": "Time displayed on the receiving device when the observation was made"
+                      },
+                      "value": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Estimated glucose value"
+                      },
+                      "realtimeValue": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Estimated glucose value presented on receiving device in realtime"
+                      },
+                      "smoothedValue": {
+                        "type": "string",
+                        "description": "Estimated glucose value presented retrospectively on receiving device after smoothing; G6 devices only"
+                      },
+                      "status": {
+                        "type": "string",
+                        "description": "Explanation of EGV record; used when the value field is outside the measuring range"
+                      },
+                      "trend": {
+                        "type": "string",
+                        "description": "General trend of EGV value movement; corresponds to trendRate field"
+                      },
+                      "trendRate": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "Rate at which glucose value is moving up or down"
+                      }
+                    }
+                  },
+                  "description": "The /egvs endpoint enables retrieval of a user's estimated glucose value (EGV) data, including trend and status information."
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get EGVs",
+        "operationId": "Get-EGVs",
+        "description": "Estimated Glucose Value (EGV) Request from Dexcom",
+        "parameters": [
+          {
+            "name": "startDate",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			"description": "The start date that will be used to for the Estimated Glucose Value (EGV) readings.",
+			"x-ms-summary": "startDate"
```

Comment:
@Flow-Joe Indentation - Some of the properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Removed tabs and added spaces